prow/plugins/bugzilla/bugzilla_test.go

```diff
@@ -1121,8 +1121,8 @@ Instructions for interacting with me using PR comments are available [here](http
 			}},
 			prs:     []github.PullRequest{{Number: base.number, Merged: true}},
 			options: plugins.BugzillaBranchOptions{StateAfterMerge: &modified}, // no requirements --> always valid
-			expectedComment: `org/repo#1:@user: An error was encountered searching for external tracker bugs for bug 123 on the Bugzilla server at www.bugzilla:
-> injected error adding external bug to bug
```

Comment:
Why did unrelated tests change?

Reply:
Because the fake client is extremely non-specific with injected errors. We simply mark a bug to inject errors for and then that error occurs for any operation (except create bug). Since the first thing we do in handle now is get the bug, we hit the "getting bug" error before the error for adding an external bug, which used to be the first error we would hit in this unit test.

We would have to change the way the fake client injects errors and make it more specific/targeted based on operation to make prevent this test from changing.